encryption-pgp/encryption-pgp.qml

```diff
@@ -20,9 +20,9 @@ QtObject {
             "default": "/usr/bin/gpg",
         },
         {
-            "identifier": "publicKey",
```

Comment:
do you need to change the identifier? this would break all existing configurations.